### ISSUE TITLE
Share access of Google cloudflare cname flattening

### ISSUE THREAD

**Issue by egen155** [biztransit/techsupport#22]
Please share access of [google cloudflare cname flattening] to this email > 
contact@hitechmain.com

**Comment by hiqsociety** [comment]
pls see attached. use z0.bczcentral.com

@egen155 
![Screenshot from 2023-09-06 12-54-09](https://github.com/biztransit/techsupport/assets/51492452/223289d2-7e35-4bd6-ae6b-b2335b2955f0)

![Screenshot from 2023-09-05 10-06-20](https://github.com/biztransit/techsupport/assets/51492452/7095bad3-2138-43b7-84d4-562591d24fb6)

**Comment by egen155** [comment]
Please share the access of  [google cloudflare cname flattening]  for the  http://aateamremisier.com/
to this email > contact@hitechmain.com

**Comment by hiqsociety** [comment]
@egen155 we dont have the direct access to cloudflare dashboard for your domain name because you already managed it yourself before. please create the cloudflare settings yourself on your own dashboard. your dns was not managed by us because you managed it yourself after domain name transferred.

pls check the attached screenshot instruction.

**Comment by egen155** [comment]
Can u share me Wordpress login details of the website > http://aateamremisier.com/

**Comment by hiqsociety** [comment]
@egen155 you need to map the domain to make the site available before we can access the wordpress login and reset the password manually for you. Please make the site up first before we reset the password.

**Comment by egen155** [comment]
![Message 1](https://github.com/biztransit/techsupport/assets/141385297/c7f0ac3c-239b-4741-b6bb-b6b61800908a)
![Message 2](https://github.com/biztransit/techsupport/assets/141385297/ced296e2-1ab6-4e78-89d5-1fd84e4a6424)


Hi, Please check the message. Thanks

**Comment by hiqsociety** [comment]
@egen155 pls set up your own DNS with the settings mentioned
pls see attached above. 
use z0.bczcentral.com as CNAME setting for aateamremisier.com

you need to create a cloudflare account to create the DNS settings. also, for SSL, pls refer to the attached image appended above too.

p.s. : previously it was already done so on your side before we have any issues. we did not manage / handle your DNS. it was all done by your side.

**Comment by egen155** [comment]
Hi, May i know what is the web hosting that host my wordpress site?

**Comment by egen155** [comment]
Hi, May i know where you bought the domain?

**Comment by egen155** [comment]
Can i get the [google cloudflare cname flattening] login?

**Comment by hiqsociety** [comment]
@egen155 
1. we use our own hosting facilities without cpanel. there's no cpanel or whatever virtual hosting panel for security reasons.
2. we already transferred to you. you should know where you transferred it to.
3. previously u used your own dns, you can sign up for a new cloudflare account or use back the dns previously used (not provided by us). after we transferred the domain name to you, 
 a) you should know which registrar was used. (we have no idea where you transferred to)
 b) you use your own dns server. (we have no idea which one u used)
 
you were using your own dns and using another registrar after it was transferred to you. we didnt do anything.

**Comment by hiqsociety** [comment]
@egen155 pls check the attached screenshot above, those information is sufficient for you to map the domain to z0.bczcentral.com as CNAME flattener for the website to show. remember to set the ssl according to cloudflare instructions above.

**Comment by egen155** [comment]
Can i have the hosting's DNS and IP that host the website content

**Comment by hiqsociety** [comment]
@egen155 
> Can i have the hosting's DNS 

Please register a new cloudflare account like you previously did.

> and IP that host the website content

@egen155 pls check the attached screenshot above, those information is sufficient for you to map the domain to z0.bczcentral.com as CNAME flattener for the website to show. 
remember to set the ssl according to cloudflare instructions above.

**Comment by egen155** [comment]
![Uploading
![WhatsApp Image 2023-09-11 at 12 31 21 PM](https://github.com/biztransit/techsupport/assets/141385297/05274e01-9054-45c6-8bdd-5f0292351ebd)
 WhatsApp Image 2023-09-12 at 10.29.51 AM.jpeg…]()


Done, but still cannot

**Comment by egen155** [comment]
![dns](https://github.com/biztransit/techsupport/assets/141385297/cdaa6727-eb6a-4060-80f0-f578ee377e7a)

![message 1 ](https://github.com/biztransit/techsupport/assets/141385297/fe9fbd0d-f958-4ec6-a9bf-7e41feb0c2b4)

**Comment by hiqsociety** [comment]
@egen155 
here, i repost again and mention again. pls set the SSL setting as below. please screenshot as desktop screen so we can see more details.

if you have set the ssl correctly, the site will show.

![Screenshot from 2023-09-11 22-34-21](https://github.com/biztransit/techsupport/assets/51492452/d1c4b077-a2d7-47ad-ab09-5f106696f971)

**Comment by egen155** [comment]
![MESSAGE ](https://github.com/biztransit/techsupport/assets/141385297/98193ed3-cc7d-4480-a53e-498a3ab54c74)

Hi, still cannot get the dns

**Comment by hiqsociety** [comment]
@egen155 can you screenshot the dns option you have done in cloudflare?

can you try "Flexible" for the ssl/tls, Encrypts traffic between the browser and Cloudflare

**Comment by hiqsociety** [comment]
@egen155 because you are using a legacy version of the old hosting platform, 
pls use this too.

![Screenshot from 2023-09-13 21-33-07](https://github.com/biztransit/techsupport/assets/51492452/592b72ca-a727-4d4e-8b0e-e75dd0b17aae)

if the issue is resolved, please close this ticket. thx

**Comment by egen155** [comment]
![image1](https://github.com/biztransit/techsupport/assets/141385297/dab2b839-a90e-4b4a-8e0c-17bd622c1d15)
![image4](https://github.com/biztransit/techsupport/assets/141385297/9321bc1f-acec-4119-aa86-89c9cbdd09f1)
![image3](https://github.com/biztransit/techsupport/assets/141385297/99bbaaee-0e13-4edc-8e05-ab9169afdd91)
![image2](https://github.com/biztransit/techsupport/assets/141385297/05de054c-3d35-42e4-9589-472591d9a550)


still cannot

**Comment by egen155** [comment]
<img width="626" alt="image" src="https://github.com/biztransit/techsupport/assets/141385297/97f9eacb-5fb6-4d91-87e5-d93aaee961c6">

**Comment by egen155** [comment]
https://dash.cloudflare.com/login

Can i confirm is the cloudfare account under your side?

**Comment by egen155** [comment]
The Cloudflare where was your website appointed? 
Where were you managing your website or dns after your service provider?

**Comment by hiqsociety** [comment]
@egen155 The original ssl is correct. which means FULL not Flexible.

Please show your DNS entry using desktop, please screenshot in desktop mode or show how you set the DNS.

the setting of cloudflare is on your side. please show what u did for your dns area.

i'm reattaching the screenshot again

![Screenshot from 2023-09-17 03-22-39](https://github.com/biztransit/techsupport/assets/51492452/6a4adeb4-3130-4a8e-a8fb-f95f26d90926)

**Comment by egen155** [comment]
![message1](https://github.com/biztransit/techsupport/assets/141385297/505b121c-5758-4b09-91f2-314374f44306)
![message2](https://github.com/biztransit/techsupport/assets/141385297/ff168933-1e39-4518-b14e-b89b15afe6a9)

**Comment by hiqsociety** [comment]
Hey @egen155,

I noticed that you've got GoDaddy as your DNS provider, but it doesn't support CNAME flattening. No worries, though! I'd recommend switching to Cloudflare or another DNS solution that does support it.

Cloudflare is a great choice, and it's super user-friendly. If you can, please go ahead and use Cloudflare for your DNS needs.

And just a quick reminder, when you're setting things up in Cloudflare, make sure to refer back to the settings we discussed earlier. The CNAME you should use is "z0.bczcentral.com" – you can find all the details in the screenshot I sent you previously.

Let me know if you have any questions or need further assistance! 😊

**Comment by hiqsociety** [comment]
@egen155 also, mys2020v2-1.lispsi.com are the new lispsi sites used in egenevent.com BUT aateamremisier.com is using old pre-2020 site so the cname to be used for dns is "z0.bczcentral.com". pls see the above screenshot and replace biomaqs.com with aateamremisier.com

**Comment by egen155** [comment]
![message](https://github.com/biztransit/techsupport/assets/141385297/bd75ec24-dd6c-4bc3-90e5-9fea4c5e7983)

**Comment by egen155** [comment]
<img width="236" alt="image" src="https://github.com/biztransit/techsupport/assets/141385297/e0b4d72c-dfb1-4bae-a3fe-98e3249a6c1e">

**Comment by hiqsociety** [comment]
@egen155 it's zero. you can copy the O or 0 if not sure and paste on google search to find too.

**Comment by egen155** [comment]
Yes it is already the same

**Comment by egen155** [comment]
but still cannot

**Comment by hiqsociety** [comment]
@egen155 
1. can you get the previous settings we have already sent by asking your previous solution provider?
2. otherwise, pls set the cloudflare name servers to cory.ns.cloudflare.com and lila.ns.cloudflare.com only to check on our side if things are ok.

**Comment by egen155** [comment]
<img width="277" alt="image" src="https://github.com/biztransit/techsupport/assets/141385297/1639f0f0-fda9-47fe-b203-c7529c40066b">

**Comment by hiqsociety** [comment]
@egen155 
https://aateamremisier.com/

the site is working correctly. your settings are wrong from your side.

can you use back your settings on cloudflare as already mentioned on github? 

![268488501-6a4adeb4-3130-4a8e-a8fb-f95f26d90926](https://github.com/biztransit/techsupport/assets/51492452/b3a52ff7-2e78-4be9-adba-58855d5e10f0)

the first screenshot already showed the proper setup.

we have already mentioned this multiple times.

**Comment by egen155** [comment]
ya  i also setup ald

**Comment by egen155** [comment]
![message1](https://github.com/biztransit/techsupport/assets/141385297/9ba50c4f-a23f-403f-bfa7-32a77ca3d784)
![message2](https://github.com/biztransit/techsupport/assets/141385297/4812a2a7-e8a8-4dbd-a61b-d9f319666054)
![message3](https://github.com/biztransit/techsupport/assets/141385297/6e2c660e-7111-4d1b-aec8-3cd52f8e081a)
![message4](https://github.com/biztransit/techsupport/assets/141385297/beed7c99-c3ba-4ee1-b150-cf325c4bbe83)
![message5](https://github.com/biztransit/techsupport/assets/141385297/2269934b-2d06-4a6c-8c7c-b86eb9335bd3)
![message6](https://github.com/biztransit/techsupport/assets/141385297/0730d69f-25e2-43e9-b500-903a8ba6fabe)
![message7](https://github.com/biztransit/techsupport/assets/141385297/6a9db1e1-325c-4e37-9497-f6311ae04efc)
![message8](https://github.com/biztransit/techsupport/assets/141385297/952e0cf6-03ae-4688-9af8-67d1d8407a2d)
![message9](https://github.com/biztransit/techsupport/assets/141385297/d05d2ae0-4ec6-4853-abdf-de33b87c5de2)
![message10](https://github.com/biztransit/techsupport/assets/141385297/8003dfa2-93eb-4e8c-876d-24a8977cf9f4)

**Comment by egen155** [comment]
these are all the screenshot from my side
please point out which part i setup incorrectly

**Comment by hiqsociety** [comment]
![Screenshot from 2023-09-21 04-05-16](https://github.com/biztransit/techsupport/assets/51492452/af1432cf-2788-4fb9-a1c9-5854ae3ae838)
@egen155 i'm going to repeat myself again and again. please use desktop mode to show proper setting.

**Comment by egen155** [comment]
ok  hold on

**Comment by hiqsociety** [comment]
@egen155 
![Screenshot from 2023-09-21 04-13-45](https://github.com/biztransit/techsupport/assets/51492452/35713fa0-b40d-4262-887e-b340fe0c93bd)

**Comment by egen155** [comment]
![WhatsApp Image 2023-09-21 at 10 03 24 PM](https://github.com/biztransit/techsupport/assets/141385297/2396cff6-5cf4-4c4f-864f-0bec7aa0dd5c)

**Comment by egen155** [comment]
i give u this one first

**Comment by egen155** [comment]
<img width="1436" alt="image" src="https://github.com/biztransit/techsupport/assets/141385297/e0883c63-c26b-4c43-80d6-4e72f2a9ad33">

https://whois.domaintools.com/aateamremisier.com

can check from here

**Comment by hiqsociety** [comment]
@egen155 pls show the setting for the cloudflare dashboard area of your dns using desktop screenshot

**Comment by egen155** [comment]
<img width="582" alt="image" src="https://github.com/biztransit/techsupport/assets/141385297/64ab1dd0-c5b3-4fab-b55f-3e64ca352e0c">

**Comment by egen155** [comment]
Who is having this cloudflare?

**Comment by hiqsociety** [comment]
@egen155 please use your own cloudflare settings. We only did test on our side and it works.

Please follow the instructions on setting for cloudflare as mentioned on this issue mentioned.

1. If you are not able to follow instructions, please screenshot the cloudflare dns setting in desktop mode. <= this is an instruction by itself we have repeated multiple times without any feedback on this issue.

2. we suspect your proxy setting on cloudflare is not enabled "orange". please check cloudflare documentation on how to set the proxy by clicking to enable orange icon.

We will disable the DNS cloudflare setting on our side as we have additional and optional special customized DNS settings on our cloudflare managed customers, only available for selected paid customers. 


3. please update back your DNS servers to use your own cloudflare. We will terminate our cloudflare test DNS instance in 3 days.

**Comment by egen155** [comment]
![message1](https://github.com/biztransit/techsupport/assets/141385297/658ced5f-43e4-4977-8df7-4e14fb08ac1c)
![message2](https://github.com/biztransit/techsupport/assets/141385297/fc34ec70-5630-463c-abf7-4bb3f4e2f89f)

**Comment by hiqsociety** [comment]
@egen155 setting is correct, should work. pls use your own cloudflare dns setting, we will delete our cloudflare test dns in 2 days. thanks.

p.s. : this has been resolved by customer with the first message as mentioned
